testing/deprecated_test.py

```diff
@@ -44,3 +44,42 @@ def test_external_plugins_integrated(testdir, plugin):
 
     with pytest.warns(pytest.PytestConfigWarning):
         testdir.parseconfig("-p", plugin)
+
+
+def test_noprintlogs_is_deprecated(testdir):
```

Comment:
You can use parametrize and have a single test here:

```python
@pytest.mark.filterwarnings("default")
@pytest.mark.parametrize('config_mode', ["ini", "cmdline"])
def test_noprintlogs_is_deprecated_ini(testdir, config_mode):
    if config_mode == "ini":
        args = ()
        testdir.makeini(
            """
            [pytest]
            log_print=False
            """
        )
    else:
        assert config_mode == "cmdline"
        args = ("--no-print-logs",)

    testdir.makepyfile(
        """
        def test_foo():
            pass
        """
    )

    result = testdir.runpytest(*args)
    result.stdout.fnmatch_lines(
        [
            "*--no-print-logs is deprecated and scheduled for removal in pytest 6.0*",
            "*Please use --show-capture instead.*",
        ]
    )
```

Reply:
Thank you! I changed the unit test to use parametrized values now.